src/robusta/integrations/webex/sender.py

```diff
@@ -0,0 +1,259 @@
+from ...core.reporting.base import Finding, FindingSeverity
+from ...core.reporting.utils import is_image, convert_svg_to_png, SVG_SUFFIX, PNG_SUFFIX
+from ...core.reporting.blocks import (
+    MarkdownBlock,
+    BaseBlock,
+    FileBlock,
+    TableBlock,
+    List,
+)
+from ...core.sinks.transformer import Transformer
+
+from PIL import Image
+from io import BytesIO
+from webexteamssdk import WebexTeamsAPI
+from fpdf import FPDF
+import os
+from enum import Enum
+
+INVESTIGATE_ICON = "🔍"
+SILENCE_ICON = "🔕"
+
+MAX_BLOCK_CHARS = 7439
+
+ADAPTIVE_CARD_VERSION = "1.2"
+ADAPTIVE_CARD_SCHEMA = "http://adaptivecards.io/schemas/adaptive-card.json"
+ATTACHMENT_CONTENT_TYPE = "application/vnd.microsoft.card.adaptive"
+
+class CardTypes(Enum):
+    ADAPTIVE_CARD = "AdaptiveCard"
+
+class FileTypes(Enum):
+    PHOTO = "PHOTO"
+    DOCUMENT = "DOCUMENT"
+
+
+class WebexSender:
+
+    """
+    Send findings to webex.
+    Parse different findings to show on Webex UI
+    """
+
+    def __init__(self, bot_access_token: str, room_id: str, cluster_name: str):
+        self.cluster_name = cluster_name
+        self.room_id = room_id
+        self.client = WebexTeamsAPI(
+            access_token=bot_access_token
+        )  # Create a client using webexteamssdk
+
+    def send_finding_to_webex(self, finding: Finding, platform_enabled: bool):
+        message, table_blocks, file_blocks, description = self._seperate_blocks(
+            finding, platform_enabled
+        )
+
+        pdf = None
+        if file_blocks:
+
+            pdf = self._create_pdf(file_blocks)
+
+        adaptive_card = self._createAdaptiveCard(
+            message_content=message, table_blocks=table_blocks, description=description
+        )
+
+        attachment = [
+            {
+                "contentType": ATTACHMENT_CONTENT_TYPE,
+                "content": adaptive_card,
+            }
+        ]
+
+        # Here text="." is added because Webex API throws error to add text/file/markdown
+        self.client.messages.create(
+            roomId=self.room_id, text=".", attachments=attachment
+        )
+
+        if pdf:
+            # Sending pdf to webex
+
+            filename = "finding.pdf"
+            pdf.output(filename, "F")
+            self.client.messages.create(roomId=self.room_id, files=[filename])
+            os.remove(filename)
+
+    def _createAdaptiveCard(
+        self, message_content, table_blocks: List[TableBlock], description
+    ):
+
+        # https://learn.microsoft.com/en-us/adaptive-cards/
```

Comment:
How do messages look with plain markdown? I am just curious because it might save some code here as well. But mostly up to you, because I haven't seen it